peerconnection_test.go

```diff
@@ -673,6 +673,125 @@ func TestSetRemoteDescriptionInvalid(t *testing.T) {
 	})
 }
 
+func TestTransceiverByRemoteAsSamePreferCodec(t *testing.T) {
```

Comment:
IMHO from test name it is not clear what exactly the test is about.
If I understand correctly test check that `WhenTransceiverCreatedByRemoteSDP_PreferredCodecShouldBeSetSameAsOnRemotePeer`.

Reply:
sounds good. let me use a clear name but not that long